.github/workflows/android.yml

```diff
@@ -0,0 +1,32 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'temurin'
+        cache: gradle
+
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build with Gradle
+      run: ./gradlew build
+
+    - name: Unit tests
+      run: ./gradlew test
+      
+    - name: Detekt validation
+      run: ./gradlew detekt
```

Comment:
Could it affects for the PR https://github.com/soygabimoreno/LosAndroides/pull/11 where the detekt task is run through gradle?

Reply:
The detekt task already exists in the master branch and in the PR you mentioned. So it doesn't affect.